pennylane/ops/op_math/prod.py

```diff
@@ -159,6 +159,7 @@ def circuit(weights):
     """
 
     _op_symbol = "@"
+    _math_op = math.prod
```

Comment:
I like this addition! Can you add an abstract base property so that anyone making a custom composite op knows to include this as well 👍🏼 

```python
    @property
    @abc.abstractmethod
    def _math_op(self) -> Callable:
        """The function used when combining the operands of the composite operator"""
```

Reply:
Done!